titus-server-master/src/main/java/com/netflix/titus/master/scheduler/constraint/V3UniqueHostConstraint.java

```diff
@@ -0,0 +1,54 @@
+package com.netflix.titus.master.scheduler.constraint;
+
+import com.netflix.fenzo.ConstraintEvaluator;
+import com.netflix.fenzo.TaskAssignmentResult;
+import com.netflix.fenzo.TaskRequest;
+import com.netflix.fenzo.TaskTrackerState;
+import com.netflix.fenzo.VirtualMachineCurrentState;
+import com.netflix.titus.master.jobmanager.service.common.V3QueueableTask;
+
+public class V3UniqueHostConstraint implements ConstraintEvaluator {
+
+    private static final Result VALID = new Result(true, null);
+    private static final Result INVALID = new Result(false, "Task from the same job already running on the agent");
+
+    @Override
+    public String getName() {
+        return "UniqueAgentConstraint";
```

Comment:
I prefer spaces in between each of the words like the others.

Reply:
I changed other to use camel case like here.